src/components/VSelect/VSelect.vue

```diff
@@ -329,7 +329,21 @@
           click: () => {
             if (!this.isActive) this.isActive = true
           },
-          keydown: this.onKeyDown // Located in mixins/autocomplete.js
+          keydown: (e) => {
+            if (!this.isActive &&
+                !this.multiple &&
```

Comment:
I don't follow, why does the DOM position matter? The menu is not showing, moving the furthest one down seems like a somewhat arbitrary choice. If you mean the last item, then it also seems somewhat arbitrary because the items are sorted alphabetically.